sleap/nn/monitor.py

```diff
@@ -423,7 +423,7 @@ def test_point(x=[0]):
 
```

Comment:
This whole section is just a convenience entrypoint for development.

Do you mind moving this into a test? (with the 20ms timeout)

Reply:
Ro you mean moving this logic to a unit test?